iogt/static/css/_base.scss

```diff
@@ -66,6 +66,8 @@ select {
 }
 
 textarea {
+  box-sizing: border-box;
```

Comment:
We've got to avoid using `border-box` for now because of device support.

The best way to achieve the result you want:

```
<div style="padding: 5px;">
    <textarea style="width: 100%"></textarea>
</div>
````